docs/components/input-field/index.md

```diff
@@ -79,6 +94,7 @@ Target the error block via `data-error`.
 ## UI States
 
 ### InputField with Label
+
```

Comment:
Some auto-format prettier markdown stuff due to whitespace - if this annoys you, you can toggle to hide whitespace in GitHub settings! It's only this file though!